src/data/ha-state-history-data.html

```diff
@@ -104,7 +103,9 @@
       this._setIsLoading(true);
 
       data.then(function (stateHistory) {
-        this._setData(stateHistory);
+        // Dom restoration is async and if graph is drown before dom is
```

Comment:
#60 has been addressed by the fix I mentioned

This fix should be removed here, it's this element's role to get the data as fast as possible. When it is being used in the history panel there is no DOM restoration happening.

If you load the data before the UI is visible, the graph will not know how much space it has to take and usually ends up taking more width than is ok.